### ISSUE TITLE
Documenting this family member in the new compendium

### ISSUE THREAD

**Issue by jh-RLI** [OpenEnergyPlatform/open-dapro#27]
It would be great if someone could document oepn-dapro in our compendium. This will help to keep an overview of all our tools, and by providing a short description, users can better understand what this tool does in the context of the OE family. 

The compendium is a simple mkdocs page hosted by github pages. It is maintained in this repo. : https://github.com/OpenEnergyPlatform/organisation
Content can be added by simply writing Markdown. 
We use https://squidfunk.github.io/mkdocs-material/  

I assume this tool could be documented on this site? What do you think?

https://github.com/OpenEnergyPlatform/organisation/blob/production/compendium/family_members/open-process-integration.md

The rendered website looks like this:
https://openenergyplatform.github.io/organisation/family_members/open-process-integration/

If you don't want to go to the trouble of adding content to the compendium, it would be nice if you could provide the descriptive text in this issue so I can add it later.

**Comment by FlorianK13** [comment]
Hi @jh-RLI - I just saw this Issue. Yes, it is probably good to document open-dapro in the collection of OE family tools. Maybe the content of the documentation page from open-dapro could be of use? https://openenergyplatform.github.io/open-dapro/

**Comment by jh-RLI** [comment]
Good idea, I will add a short text from the docs. It would be great if you could check if this is ok. I'll let you know once I've added it.
